### PR TITLE
add com.pengaru.glimmer.json

### PR DESCRIPTION
**Pull request by vcaputo**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by vcaputo** [comment]
Am new to the flatpak/flathub way of things, using this fun little gtk3 eye-candy gadget I've been poking at in spare time as an excuse to explore all this new stuff.

@chergert provided this manifest after playing with very early glimmer awhile back but I only just now got things to where it seemed worth potentially subjecting to random people.  I must admit I haven't spent any further time scrutinizing the manifest's correctness or validity, lmk if it needs any changes as it might have bitrotted it's been so long since he first created the PR adding it on the glimmer repo.

File: com.pengaru.glimmer.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "app-id" : "com.pengaru.glimmer",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.38",
```
Comment (hfiguiere) [inline com.pengaru.glimmer.json:4]: this runtime version has been EOL for quite some time.
```suggestion
    "runtime-version" : "42",
```

File: com.pengaru.glimmer.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    ],
+    "modules" : [
+        {
+            "name" : "glimmer",
+            "buildsystem" : "autotools",
+            "builddir" : false,
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://git.pengaru.com/glimmer"
```
Comment (hfiguiere) [inline com.pengaru.glimmer.json:25]: you need a tag or commit here.

File: com.pengaru.glimmer.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    ],
+    "cleanup" : [
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "glimmer",
+            "buildsystem" : "autotools",
+            "builddir" : false,
```
Comment (hfiguiere) [inline com.pengaru.glimmer.json:21]:
```suggestion
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, build com.pengaru.glimmer

**Comment by vcaputo** [comment]
Looks like I need to enable "smart" http capabilities for my git repos served via https on pengaru.com, will comment and update the manifest to address the above suggestions when I get  around to this.

Thanks.

**Comment by hfiguiere** [comment]
you can add `disable-shallow-clone: true` to the source entry

**Comment by vcaputo** [comment]
No need, setup the git-http-backend and updated the redirect to point there instead of cgit.

I think this should succeed now (actually tested here with flatpak-builder, unlike before), thanks.

**Comment by barthalion** [comment]
bot, build com.pengaru.glimmer

**Comment by barthalion** [comment]
You need to install an appdata file. https://www.freedesktop.org/software/appstream/metainfocreator/ can be used to generate it, and `org.freedesktop.appstream-glib` from Flathub to validate it.

**Comment by nedrichards** [comment]
And a desktop file if I read https://git.pengaru.com/cgit/glimmer/.git/tree/ correctly.

**Comment by vcaputo** [comment]
Thanks guys, It'll probably take me a week+ to get around to addressing those items due to travel stuffs.

**Pull request closed without merging**